apps/web/app/future/settings/(settings)/organizations/appearance/layout.tsx

```diff
@@ -1,5 +0,0 @@
-import { WithLayout } from "app/layoutHOC";
```

Comment:
`(settings)` route group already has a layout file, so this layout is an extra layer that is no longer needed. Same for other layout files I deleted.